Resources/Private/Language/locallang_db.xlf

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<xliff version="1.0" xmlns:t3="http://typo3.org/schemas/xliff">
+	<file t3:id="1415814785" source-language="en" datatype="plaintext" original="messages"
+		date="2023-01-19T20:22:32Z" product-name="wv_deepltranslate">
+		<header />
+		<body>
+			<trans-unit id="tx_wvdeepltranslate_has_translated_content" xml:space="preserve">
+				<source>Has Deepl Translated Content</source>
```

Comment:
```suggestion
				<source>Has DeepL Translated Content</source>
```
as DeepL is an own given name, the spelling should be identical to the DeepL project